recipe/meta.yaml

```diff
@@ -218,9 +217,8 @@ outputs:
         - scipy
 
     test:
-      imports:
-        - ray.rllib
       commands:
+        - python -c "import ray.rllib"
```

Comment:
What's the point of changing this away from `imports:`?

Reply:
It is left over from previous refactorings to make `gym` optional and install it via pip, which is why pip was made part of the runtime dependencies. I will remove pip and revert the testing format.